.travis-command-ea-builds.sh

```diff
@@ -41,25 +40,12 @@ function install_jdk_and_run_ea_build {
   export PATH="${JAVA_HOME}/bin:${PATH}"
 
   $JAVA_HOME/bin/java -version
-  echo "export MAVEN_OPTS='-Dmaven.repo.local=$HOME/.m2/repository -Xmx2g -XX:MaxPermSize=2048m'\" > ~/.mavenrc"
```

Comment:
The [before_install section of .travis.yml](https://github.com/eclipse/eclipse-collections/blob/master/.travis.yml#L5-L7) should still run regardless, so I think this was always redundant. The max memory is different, but doesn't need to be. MaxPermSize hasn't had any effect since Java 7.